### PR TITLE
Complete schema changes for suite-level tests and ignored suite IDs for buggy-output tests.

### PR DESCRIPTION
**Pull request by seanlip**
@eyurko PTAL

Note: Suite-level tests are now part of the task schema, but aren't integrated into the feedback flow yet. I'll do that in a separate follow-on PR after this gets merged, since the current one is getting long.

### CODE REVIEW

File: assets/questions/reverseWords.js [hunk 1]
```diff
@@ -395,10 +427,33 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
...
       messages: [
         'Note that numbers should not be treated as English words.'
       ]
     }],
+    suiteLevelTests: [{
+      testSuiteIdsThatMustPass: ['SAMPLE_INPUT', 'GENERAL_CASE', 'WHITESPACE'],
+      testSuiteIdsThatMustFail: ['CORNER_CASES'],
+      messages: [
+        [
+          'Are you happy with the code? It must give the correct answer in ',
```
Comment (eyurko) [inline assets/questions/reverseWords.js]: I don't think the first sentence is appropriate. Maybe "It seems like your code is failing on some test cases. Try stepping through some sample inputs on paper and see if your output is consistent with your expected output in all cases."

File: client/question/services/SolutionHandlerServiceSpec.js [hunk 2]
```diff
@@ -368,5 +366,79 @@ describe('SolutionHandlerService', function() {
...
+            input: 'task_1_suite_2_test_1',
+            allowedOutputs: [false]
+          }, {
+            input: 'task_1_suite_2_test_2',
+            allowedOutputs: [false]
+          }]
+        }];
+      }));
+
+      it('should check all buggy output if nothing is ignored', function(done) {
```
Comment (eyurko) [inline client/question/services/SolutionHandlerServiceSpec.js]: Nit: buggy outputs

Comment (eyurko) [reply client/question/services/SolutionHandlerServiceSpec.js]: and others below, where needed.

Comment (seanlip) [reply client/question/services/SolutionHandlerServiceSpec.js]: Done.

File: assets/questions/reverseWords.js (hunk 1 above)
Comment (seanlip) [reply assets/questions/reverseWords.js]: Done. Thank you for the catch!

**Comment by seanlip** [comment]
Thanks, @eyurko! All comments addressed. PTAL.

**Pull request merged**